spec/action.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable react/jsx-filename-extension */
```

Comment:
instead of disabling, you can try adding this into your jest.config.js/or package.json if thats where you defined your jest config too. this allows jest to test jsx files.

``` moduleFileExtensions: ['js', 'jsx']```

Reply:
I did this by naming my test files .jsx and then imported my App.jsx..., along with some configs for babel-jest. Not sure if it would work the same way for you since youre testing redux too, so just a suggestion :)